packages/fiber/src/core/stages.ts

```diff
@@ -100,7 +100,7 @@ export class FixedStage extends Stage {
   frame(delta: number, frame?: THREE.XRFrame | undefined) {
     const initialTime = performance.now()
     let substeps = 0
-    this._substepDelta = []
+    this._substepFrameTime = []
```

Comment:
I know I'm being a pest :) but can we make these `substepFrameTimes` (plural) to communicate that it's an array of values?

Reply:
I appreciate your attention to detail. ;)